chapters/en/slides/module2/module2_01.Rmd

```diff
@@ -0,0 +1,234 @@
+---
+params:
+  dynamictitle: "module2_01"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+# Effective Use of Visual Channels
+
+Notes:
+So far we have seen how to use points and lines to represent data visually.
+In this lecture,
+we will see how we can also use areas and bars for this purpose.
+Before we dive in to the code,
+let's discuss how different visual channels,
+such as position, area, etc,
+can impact how easy it is for us
+to accurately interpret the plotted data.
+
+As in many cases,
+an efficient way to learn this topic is through personal experience.
+Please follow along in the exercises on the next two slides
+to see which visual channels you find the most effective.
+
+---
+
+## Comparing sizes is easier for some geometrical objects than for others
+
+![](/module2/circle-comparison.png)
+
+Notes:
+In the image in this slide,
+try to estimate how many times larger the big circle is compared to the small one.
+
+---
+
+## Comparing sizes is easier for some geometrical objects than for others
+
+![](/module2/bar-comparison.png)
```

Comment:
This bar and square combo does not make it easy to answer this question... Could we do one where it's obviously a quarter? And then, the circle one should be a quarter? I think the contrast would be quite stark.

Reply:
I see that these slides are borrowed, but I think we could easily remake our own and still attribute where the inspiration came from.

Reply:
I have made the slides myself, but the idea is from the viz research talk I referenced and I used the same proportions they used since I thought there might be a reason behind it. I can remake them using 1/4 instead, it might be too easy to interpret the circle correctly then? Let's see.